src/avfoundation.cs

```diff
@@ -7565,6 +7624,213 @@ interface AVCaptureMetadataOutputObjectsDelegate {
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVCaptureResolvedPhotoSettings
+	{
+		[Export ("uniqueID")]
+		long UniqueID { get; }
+
+		[Export ("photoDimensions")]
+		CMVideoDimensions PhotoDimensions { get; }
+
+		[Export ("rawPhotoDimensions")]
+		CMVideoDimensions RawPhotoDimensions { get; }
+
+		[Export ("previewDimensions")]
+		CMVideoDimensions PreviewDimensions { get; }
+
+		[Export ("livePhotoMovieDimensions")]
+		CMVideoDimensions LivePhotoMovieDimensions { get; }
+
+		[Export ("flashEnabled")]
+		bool IsFlashEnabled { [Bind ("isFlashEnabled")] get; }
+
+		[Export ("stillImageStabilizationEnabled")]
+		bool IsStillImageStabilizationEnabled { [Bind ("isStillImageStabilizationEnabled")] get; }
+	}
+
+#if !MONOMAC
+
+	interface IAVCapturePhotoCaptureDelegate {}
+
+	[NoTV, NoMac, iOS (10,0)]
+	[Protocol, Model]
+	[BaseType (typeof(NSObject))]
+	interface AVCapturePhotoCaptureDelegate
+	{
+		[Export ("captureOutput:willBeginCaptureForResolvedSettings:")]
+		void WillBeginCapture (AVCapturePhotoOutput captureOutput, AVCaptureResolvedPhotoSettings resolvedSettings);
+
+		[Export ("captureOutput:willCapturePhotoForResolvedSettings:")]
+		void WillCapturePhoto (AVCapturePhotoOutput captureOutput, AVCaptureResolvedPhotoSettings resolvedSettings);
+
+		[Export ("captureOutput:didCapturePhotoForResolvedSettings:")]
+		void DidCapturePhoto (AVCapturePhotoOutput captureOutput, AVCaptureResolvedPhotoSettings resolvedSettings);
+
+		[Export ("captureOutput:didFinishProcessingPhotoSampleBuffer:previewPhotoSampleBuffer:resolvedSettings:bracketSettings:error:")]
+		void DidFinishProcessingPhoto (AVCapturePhotoOutput captureOutput, [NullAllowed] CMSampleBuffer photoSampleBuffer, [NullAllowed] CMSampleBuffer previewPhotoSampleBuffer, AVCaptureResolvedPhotoSettings resolvedSettings, [NullAllowed] AVCaptureBracketedStillImageSettings bracketSettings, [NullAllowed] NSError error);
+
+		[Export ("captureOutput:didFinishProcessingRawPhotoSampleBuffer:previewPhotoSampleBuffer:resolvedSettings:bracketSettings:error:")]
+		void DidFinishProcessingRawPhoto (AVCapturePhotoOutput captureOutput, [NullAllowed] CMSampleBuffer rawSampleBuffer, [NullAllowed] CMSampleBuffer previewPhotoSampleBuffer, AVCaptureResolvedPhotoSettings resolvedSettings, [NullAllowed] AVCaptureBracketedStillImageSettings bracketSettings, [NullAllowed] NSError error);
+
+		[Export ("captureOutput:didFinishRecordingLivePhotoMovieForEventualFileAtURL:resolvedSettings:")]
+		void DidFinishRecordingLivePhotoMovie (AVCapturePhotoOutput captureOutput, NSUrl outputFileUrl, AVCaptureResolvedPhotoSettings resolvedSettings);
+
+		[Export ("captureOutput:didFinishProcessingLivePhotoToMovieFileAtURL:duration:photoDisplayTime:resolvedSettings:error:")]
+		void DidFinishProcessingLivePhotoToMovieFile (AVCapturePhotoOutput captureOutput, NSUrl outputFileUrl, CMTime duration, CMTime photoDisplayTime, AVCaptureResolvedPhotoSettings resolvedSettings, [NullAllowed] NSError error);
```

Comment:
You are right, I initially left it because I did not see the verb diff (Processing/Recording) in that case, it makes perfect sense to remove the ToMovieFile.